app/views/layouts/application.html.erb

```diff
@@ -10,9 +10,15 @@
 
   <body>
     <nav class = "topnav">
+      <% if current_user != nil %>
```

Comment:
Nice conditional.  I know Mike doesn't like to test a negative so maybe we reverse this in a refactor